src/audio/dai.c

```diff
@@ -63,7 +64,8 @@ struct dai_data {
 	struct dai *dai;
 	struct dma *dma;
 	uint32_t frame_bytes;
-	int xrun;		/* true if we are doing xrun recovery */
+	bool xrun;		/* true if we are doing xrun recovery */
```

Comment:
Didn't we want to remove xrun recovery at all from fw?

Reply:
I think the comment is wrong and it's only used to notify the DAI we are in xrun, but I haven't verified the logic behind it.